torchaudio/models/_wavernn.py

```diff
@@ -1,105 +1,325 @@
-    WaveRNN is based on the paper "Efficient Neural Audio Synthesis". Nal Kalchbrenner, Erich Elsen,
-    Karen Simonyan, Seb Noury, Norman Casagrande, Edward Lockhart, Florian Stimberg, Aaron van den Oord,
-    Sander Dieleman, Koray Kavukcuoglu. arXiv:1802.08435, 2018.
+    r"""MelResNet layer based on a stack of ResBlocks.
 
     Args:
-        res_blocks: the number of ResBlock in stack (default=10).
-        input_dims: the number of input sequence (default=100).
-        hidden_dims: the number of compute dimensions (default=128).
-        output_dims: the number of output sequence (default=128).
-        pad: the number of kernal size (pad * 2 + 1) in the first Conv1d layer (default=2).
+        n_res_block: the number of ResBlock in stack (default=10)
+        n_freq: the number of bins in a spectrogram (default=128)
+        n_hidden: the number of hidden dimensions (default=128)
+        n_output: the number of output dimensions (default=128)
+        kernel_size: the number of kernel size in the first Conv1d layer (default=5)
 
     Examples::
-        >>> melresnet = _MelResNet(res_blocks=10, input_dims=100,
-                                   hidden_dims=128, output_dims=128, pad=2)
-        >>> input = torch.rand(10, 100, 512)
+        >>> melresnet = _MelResNet(n_res_block=10, n_freq=128, n_hidden=128,
+                                   n_output=128, kernel_size=5)
+        >>> input = torch.rand(10, 128, 512)
         >>> output = melresnet(input)
     """
 
-    def __init__(self, res_blocks: int = 10,
-                 input_dims: int = 100,
-                 hidden_dims: int = 128,
-                 output_dims: int = 128,
-                 pad: int = 2) -> None:
+    def __init__(self,
+                 n_res_block: int = 10,
+                 n_freq: int = 128,
+                 n_hidden: int = 128,
+                 n_output: int = 128,
+                 kernel_size: int = 5) -> None:
         super().__init__()
 
-        kernel_size = pad * 2 + 1
-        ResBlocks = []
-
-        for i in range(res_blocks):
-            ResBlocks.append(_ResBlock(hidden_dims))
+        ResBlocks = [_ResBlock(n_hidden) for _ in range(n_res_block)]
 
         self.melresnet_model = nn.Sequential(
-            nn.Conv1d(in_channels=input_dims, out_channels=hidden_dims, kernel_size=kernel_size, bias=False),
-            nn.BatchNorm1d(hidden_dims),
+            nn.Conv1d(in_channels=n_freq, out_channels=n_hidden, kernel_size=kernel_size, bias=False),
+            nn.BatchNorm1d(n_hidden),
             nn.ReLU(inplace=True),
             *ResBlocks,
-            nn.Conv1d(in_channels=hidden_dims, out_channels=output_dims, kernel_size=1)
+            nn.Conv1d(in_channels=n_hidden, out_channels=n_output, kernel_size=1)
         )
 
     def forward(self, x: Tensor) -> Tensor:
-        r"""Pass the input through the _MelResNet layer.
-
+        r"""
```

Comment:
What's your rational behind removing the original short description, `Pass the input through the _MelResNet layer.`??

Reply:
When I removed it, I was thinking that this description repeats because the forward function itself means passing the input through this function. I am okay with it and this sentence has been added in each function.